config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main-upgrades.yaml

```diff
@@ -116,9 +116,9 @@ periodics:
         - "./scripts/ci-e2e.sh"
       env:
         - name: KUBERNETES_VERSION_UPGRADE_FROM
-          value: "stable-1.26"
-        - name: KUBERNETES_VERSION_UPGRADE_TO
           value: "stable-1.27"
+        - name: KUBERNETES_VERSION_UPGRADE_TO
+          value: "stable-1.28"
```

Comment:
for etcd and coredns version we usually follow what CAPI does, in this case looks like there are newer versions we can use for 1.28 https://github.com/kubernetes/test-infra/blob/f6900f9909c78cacc798cd42df099cbdf9c11646/config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml#L250

Reply:
Good catch. Just added it!